scripts/c_api_tests.sh

```diff
@@ -28,6 +31,12 @@ do
         "--gpu" )
             WITH_FEATURE_GPU="ON"
             ;;
+
+        "--forward-compat" )
+            shift
```

Comment:
Is `shift` needed here?

Reply:
yes it's taking an ON or OFF value in the Makefile